### PR TITLE
Adds ocm-advisor

### PR DESCRIPTION
**Pull request by AllenBW**
fixes https://issues.redhat.com/browse/CCXDEV-5083

### CODE REVIEW

**Comment by AllenBW** [comment]
cc @gkarat

File: main.yml [hunk 1]
```diff
@@ -644,6 +646,20 @@ openshift:
...
 
+ocp-advisor: 
+  title: Advisor
+  source_repo: https://github.com/RedHatInsights/ocp-advisor-frontend
+  deployment_repo: https://github.com/RedHatInsights/ocp-advisor-frontend-build
+  frontend:
+    module: ocp-advisor#./RootApp
+    isHidden: true
+    paths:
+      - /openshift/advisor
```
Comment (gkarat) [inline main.yml]: There was a requirement for the new Advisor to run under the `/openshift/insights/advisor` URL, so it seems we need to rewrite it to the suggested format :)

File: main.yml [hunk 2]
```diff
@@ -644,6 +646,20 @@ openshift:
...
+    sub_apps:
```
Comment (gkarat) [inline main.yml:658]: Could you also add the `clusters`, please?

File: main.yml [hunk 3]
```diff
@@ -644,6 +646,23 @@ openshift:
 
   top_level: true
 
+ocp-advisor: 
+  title: Advisor
+  source_repo: https://github.com/RedHatInsights/ocp-advisor-frontend
+  deployment_repo: https://github.com/RedHatInsights/ocp-advisor-frontend-build
+  frontend:
+    module: ocp-advisor#./RootApp
+    isHidden: true
```
Comment (gkarat) [inline main.yml:655]: Just a question to make sure: when `isHidden` is set to true, then the app is not available in all envs within its path, right?

Comment (AllenBW) [reply main.yml:655]: @gkarat ishidden ensures the entry does not show up in the lnav this pr only adds a csc record for ci-beta

**Comment by gkarat** [comment]
LGTM! 🎉  As far as I can see

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**